pkgs/applications/science/physics/crystfel/default.nix

```diff
@@ -143,10 +137,10 @@ let
     pname = "HDF5-External-Filter-Plugins";
     version = "0.1.0";
     src = fetchFromGitHub {
-      owner = "nexusformat";
+      owner = "spanezz";
```

Comment:
The only difference is one commit https://github.com/nexusformat/HDF5-External-Filter-Plugins/compare/master...spanezz:HDF5-External-Filter-Plugins:master so we can fetchpatch that

Reply:
See comment below.